rc/extra/kitty.kak

```diff
@@ -1,8 +1,13 @@
-hook global KakBegin .* %sh{
+declare-option -docstring %{window type that kitty creates on new and repl calls (kitty|os)} str kitty_window_type kitty
```

Comment:
I dont really get the interest of `os` window type, doesnt it boils down to doing the same as x11.kak and x11-repl.kak ? Once kitty does not do window management anymore, why go through kitty.kak instead of x11.kak (so a user can opt to use x11.kak by manually overriding the aliases).

Reply:
I believe it is faster to spawn, kitty reuses some resources when doing so

Reply:
Also, if someone is using kitty on Wayland or macOS they couldn't go the X11 route, but kitty could offer that behaviour.